packages/driver/src/cross-origin/patches/cookies.js

```diff
@@ -62,7 +121,7 @@ export const patchDocumentCookie = (Cypress, window) => {
 
```

Comment:
I think we need the full `hostname` here since `Cypress.Location` omits the subdomain. If we have cookies set on `www.example.com` and we call the automation API to get cookies for `.example.com`, are the cookies from `www.example.com` omitted? 

UPDATE: looks like cookies from `.www.example.com` are included with super domain `.example.com`. My guess is that this would also mean that other subdomains might get applied to `document.cookie` when we might not want them to, like `app.example.com`.

If we inject the AUTWindow into the patch, we should be able to use `window.location.hostname` to get the domain with the subdomain in it.

Reply:
lucky for us, once we move the patch to the injection window, will be the AUT window!

Reply:
Now using [`location.hostname` instead](https://github.com/cypress-io/cypress/pull/23643/files#diff-0e98d845aa0ebdbe6c8b598f18dc81cc8f0ea4c09e7894ea32000e0ba4d116daR59).